openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-autonomic/src/tools/Autonomic/Tabs/Triggers/TriggerOperand.vue

```diff
@@ -279,6 +279,7 @@ export default {
         target: event.targetName,
         packet: event.packetName,
         item: event.itemName,
+        // TODO: event.valueType === 'RAW'
```

Comment:
TODO?

Reply:
I don't want to dig into Autonomic right now but it should probably use the TargetPacketItem native valueType rather than implement it's own